nomad/job_endpoint_hook_consul_ce.go

```diff
@@ -28,7 +29,11 @@ func (h jobConsulHook) Validate(job *structs.Job) ([]error, error) {
 	}
 
 	for _, group := range job.TaskGroups {
+
+		partition := ""
```

Comment:
nit: change this to groupPartition, so it's a little easier to understand the comparison on L36. It took me a moment to figure out what was happening.